bc_obps/registration/schema/v2/transfer_event.py

```diff
@@ -55,21 +55,56 @@ class Meta:
         fields = ['effective_date']
 
 
+FacilitiesNameType = List[Dict[str, Union[str, UUID]]]
+
+
 class TransferEventOut(ModelSchema):
     transfer_entity: str
+    from_operator: str = Field(alias="from_operator.legal_name")
+    from_operator_id: UUID = Field(alias="from_operator.id")
+    to_operator: str = Field(alias="to_operator.legal_name")
+    operation_name: Optional[str] = Field(None, alias="operation.name")
+    from_operation: Optional[str] = Field(None, alias="from_operation.name")
+    from_operation_id: Optional[UUID] = Field(None, alias="from_operation.id")
+    to_operation: Optional[str] = Field(None, alias="to_operation.name")
+    # This is only required to display the existing facilities in the frontend in the format "Facility (lat, long)"
+    existing_facilities: FacilitiesNameType
 
     class Meta:
         model = TransferEvent
-        fields = [
-            'from_operator',
-            'to_operator',
-            'effective_date',
-            'operation',
-            'from_operation',
-            'to_operation',
-            'facilities',
-        ]
+        fields = ['status', 'effective_date', 'operation', 'facilities']
 
     @staticmethod
     def resolve_transfer_entity(obj: TransferEvent) -> str:
         return "Facility" if obj.facilities.exists() else "Operation"
+
+    @staticmethod
+    def resolve_existing_facilities(obj: TransferEvent) -> FacilitiesNameType:
+        def format_facility_name(facility: Facility) -> str:
+            if (
+                facility.latitude_of_largest_emissions is not None
+                and facility.longitude_of_largest_emissions is not None
+            ):
+                return f"{facility.name} ({format_decimal(facility.latitude_of_largest_emissions, 1)}, {format_decimal(facility.longitude_of_largest_emissions, 1)})"
```

Comment:
0.1 degrees of latitude is like 11km, so possibly we could end up with facilities that have the same coordinates if they're close to each other? (Or maybe they're always far apart and one decimal is fine, do you know @zoeyli-46?)

Reply:
I'm going to use two decimal places for now.